<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionDialog</class>
 <widget class="QDialog" name="MissionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick Source &amp; Target</string>
  </property>
  <widget class="QLabel" name="label_map">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Resources/Tools/map.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="label_position_target">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_dir_source">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="buttonLoop">
      <property name="text">
       <string>Loop</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="buttonReset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="buttonSource">
      <property name="text">
       <string>Pick</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_dir_target">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="buttonOK">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_position_source">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Source</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Target</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonReset</tabstop>
  <tabstop>buttonLoop</tabstop>
  <tabstop>buttonSource</tabstop>
  <tabstop>buttonOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
